client/src/pages/NonConformities.tsx

```
@@ -1,340 +1,151 @@
-import { ConnectionStatus } from "@/components/ConnectionStatus";
-import { PhotoUpload } from "@/components/PhotoUpload";
-import { VoiceInput } from "@/components/VoiceInput";
+import { Button } from "@/components/ui/button";
+import { useVistoriaStore } from "@/stores/vistoriaStore";
+import { NaoConformidadesChecklist, type AvailableNonConformity, type SelectedNonConformity } from "@/components/forms/NaoConformidadesChecklist";
 import { useToast } from "@/hooks/use-toast";
-import { useInspection } from "@/hooks/useInspection";
-import type { NonConformity } from "@shared/schema";
+import type { NonConformity as StoreNonConformity } from "@shared/schema"; // Actual store type
+
+import type { NonConformity as StoreNonConformity, PhotoRepresentation } from "@/stores/vistoriaStore"; // Actual store type
```

Biome (1.9.4)

[error] Shouldn't redeclare 'StoreNonConformity'. Consider to delete it or rename it.

'StoreNonConformity' is defined here:


(lint/suspicious/noRedeclare)